checkov/secrets/runner.py

```diff
@@ -72,6 +72,9 @@
     'Twilio API Key': 'CKV_SECRET_18',
     'Hex High Entropy String': 'CKV_SECRET_19'
 }
+
+ENTROPY_CHECK_IDS = ('CKV_SECRET_6', 'CKV_SECRET_19')
```

Comment:
So basically this means that anyone adding a new check which uses entropy has to remember/know that this list exists. Is there a place or something you have to inherit to create a new secret check?
If we have something like that, we can add an API for understanding wether it's an entopy check or not

Reply:
1. Don't think that we will add a new entropy.
2. There is no place to inherit from.

Reply:
Makes very sense, but no place to inherit from.